spec/wavesurfer.spec.js

```diff
@@ -321,9 +321,9 @@ describe('WaveSurfer/playback:', function() {
     });
 
     /** @test {WaveSurfer#exportPCM} */
-    it('return Promise with PCM data formatted using JSON.stringify', function(done) {
+    it('return Promise with PCM data', function(done) {
         wavesurfer.exportPCM().then(pcmData => {
-            expect(pcmData).toBeNonEmptyString();
+            expect(pcmData).to.be.a('promise');
```

Comment:
Isn't it supposed to be an array of numbers?

Reply:
it's returned as a promise so it can be an array or a blob, I could add two checks one with the blob and the other just array.

![image](https://user-images.githubusercontent.com/5193884/107133035-630a0300-68b2-11eb-8601-56b56ad29e39.png)

Reply:
sounds good.

Reply:
please also add a test for case where wavesurfer.load wasn't called yet (and peaks are empty).

Reply:
> Isn't it supposed to be an array of numbers?

Agreed. It should always return an array IMO (and not a blob).